CONTRIBUTING.md

```diff
@@ -16,7 +16,6 @@
 - `yarn dev:app` to develop the **app** with local dev endpoints ([egghead-rails](https://github.com/eggheadio/egghead-rails) needs to be running locally)
   - [localhost:2000](http://localhost:2000) to view the **app**
   - `yarn dev:app:staging` to develop the **app** with staging endpoints
-  - `yarn dev:app:prod` to check the **app** with prod endpoints
```

Comment:
Remove this as nobody uses it and it probably isn't smart to develop with prod data